### PR TITLE
Add BLOCKCHAIN_CONNECTOR env var and add evmconnect to manifest

### PR DESCRIPTION
**Pull request by peterbroadhurst**
Signed-off-by: Peter Broadhurst <peter.broadhurst@kaleido.io>

### CODE REVIEW

**Comment by peterbroadhurst** [comment]
I've merged https://github.com/hyperledger/firefly/pull/948 into this

File: manifest.json [hunk 1]
```diff
@@ -1,13 +1,23 @@
 {
+  "evmconnect": {
+    "image": "ghcr.io/hyperledger/firefly-evmconnect",
+    "tag": "v0.1.0-20220810-2",
+    "sha": "bf580fd1698d9d2665a134d394cdb28e549b9e573d5f11b51dbe117ac1683ef9"
+  },
```
Comment (nguyer) [inline manifest.json:6]: Looks like we ended up with duplicate entries here. Probably due to merging various PRs together and the fact that I also added this in another branch. Sorry.

File: manifestgen.sh [hunk 2]
```diff
@@ -42,7 +42,9 @@ CLI_SECTION=$(cat manifest.json | jq .cli)
 rm -f manifest.json
 
 SERVICES=(
+    "evmconnect"
     "ethconnect"
+    "evmconnect"
```
Comment (nguyer) [inline manifestgen.sh:47]: One of these `evmconnect` entries should be removed

**Review by nguyer — APPROVED** [review]
Looks good. One minor manifest issue to clean up, but otherwise good to merge.

File: manifestgen.sh (hunk 2 above)
Comment (peterbroadhurst) [reply manifestgen.sh:47]: will sort that now - might address the test coverage gap as I have to go round again

**Pull request merged**